api/all.yaml

```diff
@@ -21,6 +21,14 @@ tags:
       Functions implement User-Defined Functions (UDFs) in Unity Catalog.
       The function implementation can be a SQL expression/query or a python function, and it can be invoked wherever a table reference is allowed in a query.
       In Unity Catalog, a function resides at the same level as a table, so it can be referenced with the form __catalog_name__.__schema_name__.__function_name__.
+  - name: Models
+    description: |
+      Models are machine learned models that are stored within Unity Catalog.  They are comprised of all versions of a single model that a user wishes to keep.
+      A model resides in the third layer of Unity Catalog’s three-level namespace, and can be referenced with the form __catalog_name__.__schema_name__.__model_name__.
+  - name: ModelVersions
+    description: |
+      Model versions contain all artifact files associated with an individual machine learned model.  They are associated with a single parent model and have an integer identifier, indicating the version number of the model.
+      Model versions are referenced via model URI's using the Unity Catalog three-level namespace of the registered model and the version identifier.
```

Comment:
Example of the URI format would help here

Reply:
There are two spaces here, you can delete one space:
`model registry.  They are comprised`
->
`model registry. They are comprised`

Same for:
`machine learned model.  They are associated`
->
`machine learned model. They are associated`